Localizations/webroot/localization_webroot-bn.po

```diff
@@ -449,6 +449,10 @@ msgstr ""
 msgid "Complete log"
 msgstr ""
 
+#: templates/backup-result/box.html:40
+msgid "Copy log"
+msgstr ""
+
```

Comment:
I believe the `.po` and `.pot` files are automatically created when we update the translations prior to a release, so we do not need to modify them here.

Reply:
Ok, I removed all the changes for .po and .pot files, should be fine.